app/views/pages/GUIDE.markdown

```diff
@@ -95,6 +96,36 @@ Some easy to score electives:
  - 0341 110 Introduction to Economics I
  - 0241 111 Introduction to Economics II
  - COMP 2067 Programming for Beginners
+ 
+Jose's recommended course sequence for the first two years of University. First two years of University most CS students take the same courses as CS doesn't have many electives. I also recommend you take mandatory courses as third and fourth year you are going to want to have electives to take fun CS courses. 
+
+#### First Semester
+COMP100 Key Concepts in comp sci
```

Comment:
Can you add the new trailing `0` for these course codes like you did for the fourth semester